spec/contivModel2raml.rb

```diff
@@ -32,8 +37,8 @@
 
       if p["type"] == "array"
         p["items"] = {
-		"type" => value["items"]
```

Comment:
I didn't remove any of your hash keys, I just fixed the indentation on everything to use two spaces (standard for Ruby code)